packages/wonder-blocks-modal/components/modal-header.test.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import React from "react";
+import {shallow} from "enzyme";
+
+import expectRenderError from "../../../utils/testing/expect-render-error.js";
+import ModalHeader from "./modal-header.js";
+
+describe("ModalHeader", () => {
+    test("using only `subtitle` should render the header", () => {
```

Comment:
We should probably also have test cases for `header` only and `header and breadcrumbs`.